src/php/config/php.ini

```diff
@@ -1769,11 +1769,16 @@ opcache.enable=1
 ; Determines if Zend OPCache is enabled for the CLI version of PHP
 ;opcache.enable_cli=0
 
+# Set default values for: 
+# opcache.memory_consumption
+# opcache.interned_strings_buffer
+# See https://github.com/nextcloud-snap/nextcloud-snap/issues/2024
+
 ; The OPcache shared memory storage size.
-;opcache.memory_consumption=128
+opcache.memory_consumption=128
 
 ; The amount of memory for interned strings in Mbytes.
-;opcache.interned_strings_buffer=8
+opcache.interned_strings_buffer=8
```

Comment:
The rubber doesn't seem to be meeting the road here: no matter what the user sets, these values will always be 128/8 until you start using the values that they set. We use [environment variables](https://github.com/nextcloud-snap/nextcloud-snap/blob/a5868d9bc03315cab14bdab6e29e5efeaf8c78e0/src/php/config/php.ini#L430) for that [elsewhere](https://github.com/nextcloud-snap/nextcloud-snap/blob/a5868d9bc03315cab14bdab6e29e5efeaf8c78e0/src/php/bin/run-php#L8).

Reply:
ok - do I just need to comment those out again? Will user input uncomment and set the new values?

Reply:
Take a look at the links I gave there-- you'll need to define environment variables holding the values set by the user, and then use those environment variables here in the config. Don't forget the php-fpm config as well.